app/views/partials/_header.html.erb

```diff
@@ -1,3 +1,5 @@
 <header>
   <h1><%= link_to "BLOG APP", root_path%></h1>
+
+  <%= link_to "Sign Up"%>
```

Comment:
- [ ] On your header, this sign-up link does not link to your sign-up page. Kindly make sure that a user can sign up and login into your app.